app/src/main/java/de/westnordost/streetcomplete/quests/oneway_suspects/AddSuspectedOneway.kt

```diff
@@ -39,6 +40,8 @@ class AddSuspectedOneway(
     override val hasMarkersAtEnds = true
     override val isSplitWayEnabled = true
 
+    override val questTypeAchievements = emptyList<QuestTypeAchievements>()
```

Comment:
Maybe add to car achievement?

However, the name is included in the quest aliases, so this will probably always be counted as `AddOneway` anyways, right?